tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/BootstrapTasks/GetBuildTools.cs

```diff
@@ -129,10 +130,14 @@ public string RemoteRootDir
                         else
                         {
                             string url = remoteUri.ToString();
-                            if (url.ToLower().Contains("readme.md"))
+                            Regex regex = new Regex("readme.md", RegexOptions.IgnoreCase);
+                            if (regex.IsMatch(url))
                             {
-                                url = url.ToLower().Replace("readme.md", "");
-                                _remoteRootDir = url;
+                                _remoteRootDir = regex.Replace(url, "");
+                            }
+                            else
+                            {
+                                BuildToolsLogger.LogException(new Exception(string.Format("Url {0} does not contain readme.md", url)));
```

Comment:
@dsgouda add a test for this scenario, this way we know this code path works the way we want it to work.

Reply:
Good catch.